src/components/ec-toaster/ec-toaster.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div
+    v-if="messages.length > 0"
+    class="ec-toaster"
+  >
+    <transition-group
+      ref="list"
+      name="list"
+      tag="ul"
+      class="ec-toaster__list"
+      appear
```

Comment:
A leftover from my PoC. `appear` doesn't work on transition-group. You can remove that.